example/provider-local/garden/base/controller-registration.yaml

```diff
@@ -13,6 +13,8 @@ apiVersion: core.gardener.cloud/v1beta1
 kind: ControllerRegistration
 metadata:
   name: provider-local
+  annotations:
```

Comment:
You might have to add it here, https://github.com/gardener/gardener/blob/aa2f8c0cd392420f3a74f33d5098478a66fad80d/example/provider-local/garden/base/kustomization.yaml#L16-L37